convex/users.ts

```
@@ -17,20 +23,27 @@ export const createOrUpdateUser = mutation({
 
     if (existingUser) {
       // Update existing user
-      await ctx.db.patch(existingUser._id, {
+      const updateData: any = {
         email: args.email,
         name: args.name,
         avatar: args.avatar,
-      })
+      }
+      
```

ESLint

[error] Delete `······`

(prettier/prettier)